api/all.yaml

```diff
@@ -722,6 +748,268 @@ paths:
           content:
             application/json:
               schema: {}
+  /models:
+    post:
+      operationId: createModel
```

Comment:
Should this be CreateRegisteredModel to match the name of the UC REST API + clarify that it's a registered model and not a model version? And similar comment for "summary" etc fields

Reply:
So this part was not clear that the operationId was anything more than the UC OSS implementation of the handler for this endpoint.  I can change it but it wasn't obvious that we wanted to keep the same nomenclature of `RegisteredModel` vs `Model`

Reply:
I'm going to go ahead and rename.